src/test/java/org/polystat/PolystatITCase.java

```diff
@@ -129,4 +151,23 @@ private static String exec(final String... cmds) throws Exception {
             .replaceFirst("Picked up .*\n", "");
     }
 
+    /**
+     * Write resource contents to a new file under the given path.
+     * @param path Directory path for a new file.
+     * @param name Name of a new file.
+     * @param resource Relative path of a resource to be copied.
+     * @throws Exception IOException - write operation failed, Exception - copying failed.
+    */
+    private static void writeFile(
+        final Path path,
+        final String name,
+        final String resource
+    ) throws Exception {
+        Files.write(
```

Comment:
@silverCase We can use `TeeInput` & `OutputTo` from Cactoos library. This way we can get rid of static methods.

Reply:
Okay.